src/MCMCMonitorDataManager/MCMCMonitorData.ts

```diff
@@ -350,3 +380,8 @@ const invalidateStats = <T extends SequenceStatsDict | VariableStatsDict>(statsD
 
     return statsDict
 }
+
+export {
+    appendData as appendData_TEST, chainsWereUpdated as chainsWereUpdated_TEST,
+    computeEffectiveWarmupIterations as computeEffectiveWarmupIterations_TEST, doChainUpdate as doChainUpdate_TEST, invalidateStats as invalidateStats_TEST
+}
```

Comment:
Exporting these just for testing is messy... but they merit testing (after all, that's where I found a bug), and it would be painfully constrained to try to test them through the public interface. I had originally written this with the tests for private functions in the same source file (so that they would be visible in scope) but at least in jest that led to compilation errors for the actual software.

Reply:
I think that's a good idea to append _TEST to those exports.